meraki_organization.tf

```diff
@@ -2,26 +2,52 @@ locals {
   organizations = flatten([
     for domain in try(local.meraki.domains, []) : [
       for org in try(domain.organizations, []) : {
-        organization_name = org.name,
+        organization_name = org.name
+        managed           = try(org.managed, true)
```

Comment:
Should we make the default `false` instead (changing the existing behavior)?

Should we move the default to the defaults file?

Reply:
Default should be `true` (to align with other architectures) and should be taken from defaults file.